packages/avatar-group/src/vaadin-avatar-group.js

```diff
@@ -358,7 +357,7 @@ class AvatarGroup extends ElementMixin(ThemableMixin(PolymerElement)) {
     }
   }
 
-  /** @private */
+  /** @protected */
   _onResize() {
     this.__debounceResize = Debouncer.debounce(this.__debounceResize, timeOut.after(0), () => {
```

Comment:
Done

Reply:
Looked into adding the debouncer but seems it wouldn't improve the performance here because the `ResizeObserver` is already kind of debounced internally:
```js
element.style.width = '100px';
// Force a reflow
console.log(element.getBoundingClientRect().width);

element.style.width = '200px';
// Force a reflow
console.log(element.getBoundingClientRect().width);
```
still only produces 1 resizeobserver invocation.